### PR TITLE
es6-classes - Jared Jenvey

### PR DESCRIPTION
**Pull request by JCJenvey**
es6-classes1
es6-classes2

### CODE REVIEW

File: es6-classes/square.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+
+class Square extends Shape {
+  constructor(area, circumference, width) {
```
Comment (scrumptiousdev) [inline es6-classes/square.js]: Per instruction:
Write its constructor to take the **square's width as a parameter** (and nothing else)

File: es6-classes/circle.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+
+class Circle extends Shape {
+  constructor(area, circumference, radius) {
```
Comment (scrumptiousdev) [inline es6-classes/circle.js]: Per instruction:
Write its constructor to take the **circle's radius as a parameter** (and nothing else)

**Review by scrumptiousdev — CHANGES_REQUESTED** [review]

File: es6-classes/square.js (hunk 1 above)
Comment (JCJenvey) [reply es6-classes/square.js]: es6-classes1

File: es6-classes/circle.js (hunk 2 above)
Comment (JCJenvey) [reply es6-classes/circle.js]: es6-classes2

**Review by scrumptiousdev — APPROVED** [review]
Approved! :)

**Pull request merged**